torch/__init__.py

```diff
@@ -917,6 +917,59 @@ def is_warn_always_enabled():
     """
     return _C._get_warnAlways()
 
+
```

Comment:
docs here would be nice, but the most important missing thing here is an indication that these need to be synchronized with their C++ equivalents.

Reply:
Added some docs and comment about keeping them consistent with C++